### PR TITLE
Add except_infobars_digest256

### PR DESCRIPTION
**Pull request by ckolos**
- assumes that the repo for the infobars list content is the same as
the flash list content; also assumes filipec has a copy of this file in
his branch. DO NOT MERGE UNTIL THIS IS THE CASE, lest we get failed
list-creation alerts ;)

### CODE REVIEW

File: stage.ini [hunk 1]
```diff
@@ -103,6 +103,11 @@ output=except-flashsubdoc-digest256
 s3_key=plugin-blocklist/except-flashsubdoc-digest256
 blocklist=https://raw.githubusercontent.com/felipc/shavar-plugin-blocklist/master/flashsubdocexceptions.txt
 
+[infobar-exceptions]
+output=except-infobars-digest256
+s3_key=blocklist/except-infobars-digest256
+blocklist=https://raw.githubusercontent.com/felipc/shavar-plugin-blocklist/master/infobarsexceptions.txt
```
Comment (groovecoder) [inline stage.ini]: Should we merge https://github.com/mozilla-services/shavar-plugin-blocklist/pull/13 and update this line to point to `mozilla-services/shavar-plugin-blocklist/master/infobarexceptions.txt` or just point it to `bytesized/shavar-plugin-blocklist/infobar/infobarexceptions.txt` ?

Comment (groovecoder) [inline stage.ini]: Also the file-name in @bytesized's branch is `infobar.txt`, not `infobarexceptions.txt`

**Comment by groovecoder** [comment]
This also needs https://github.com/mozilla-services/shavar-list-creation/pull/46 so the list-creation script will actually create the safebrowsing list out of the `.txt` file.

File: stage.ini (hunk 1 above)
Comment (ckolos) [inline stage.ini]: I would prefer the name to be consistent and specific. It's just going to cause a problem later if we're not.

In that repo, "flash.txt" is for *blocking* flash; it follows reason that infobars.txt would *block* infobars, not exception them.

Comment (bytesized) [inline stage.ini]: File was renamed in my PR.

Comment (ckolos) [inline stage.ini]: +1

Comment (ckolos) [inline stage.ini]: I'll change the path to point at the main repo file; if we need to change it later, easy enough to do so.

**Comment by groovecoder** [comment]
I ran my `lists2safebrowsing.py` locally with this `.ini` file and it worked fine on the infobars output ...
```
Plugin blocklist(infobar-exceptions): publishing 0 items; file size 18
```

**Pull request merged**